collections/Bodl/MS_Bodl_860.xml

```diff
@@ -62,102 +93,1206 @@
+                           <explicit defective="true" xml:lang="la"><locus>(fol. 93vb)</locus> Ecclesia semper est <note>catchword adds:</note> inter malleum et incudem</explicit>
+                        </msItem>
+                        <bibl>Excerpt from fol. 9r: Joseph W. Goering, <title>William de Montibus (c. 1140–1213): The Schools and the Literature of Pastoral Care</title>, Studies and Texts 108 (Toronto: Pontifical Institute of Mediaeval Studies, 1992), 497–503. </bibl>
                      </msItem>
                   </msContents>
                   <physDesc>
                      <objectDesc form="codex">
                         <supportDesc material="perg">
                            <support>parchment</support>
+                           <extent>85 leaves</extent>
+                           <foliation>Along with modern foliation, numbered i–lxxxiiij (skipping fol. 53, blank) by the cursive hand writing the index added on fols. 207–208.</foliation>
+                           <collation>
+                              <signatures>The original scribe numbered the gatherings in ink at the bottom of the last page of each gathering (·i· to ·xi·); another hand has also numbered the first page of the first three gatherings (·i· to ·iii·).</signatures>
+                              <catchwords>Catchwords in ink by the original scribe, at the bottom of the final page of each gathering in the right-hand corner.</catchwords>
+                              <secFol>dici pax patris</secFol>
+                           </collation>
                         </supportDesc>
+                        <layoutDesc>
+                           <layout columns="2" ruledLines="45 49"><dimensions type="written" unit="mm">
+                                 <height quantity="190">190</height>
+                                 <width min="137" quantity="141">137–41</width>
+                              </dimensions> Ruled in pencil; frame and individual lines pricked on all sides of page. Line spacing 4–4.5 mm.</layout>
+                           <layout columns="2" ruledLines="60"><dimensions type="written" unit="mm">
+                                 <height max="192" min="190">190–92</height>
+                                 <width max="144" min="141">141–44</width>
+                              </dimensions><locus>(Gathering 11 (fols. 86 to 93))</locus> is written with tighter line spacing (2.5-3.5 mm). </layout>
+                        </layoutDesc>
                      </objectDesc>
+                     <handDesc>
+                        <handNote script="textualisNorthern">Gothic textualis, first quarter of the twelfth century, with several shifts in hand over the course of the text. Some of these appear to be the same as those responsible for the first part of the manuscript.</handNote>
```

Comment:
(here and above)- the date seems early for textualis: would protogothic be more appropriate?